src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -91,15 +91,15 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
 
         IndexLogEntry.create(
           indexConfig.indexName,
-          CoveringIndex(
-            CoveringIndex.Properties(
-              CoveringIndex.Properties
+          HyperSpaceIndex.HashPartitionIndex(
+            HyperSpaceIndex.Properties.HashPartition(
```

Comment:
What's the correct casing here? `Hyperspace` or `HyperSpace`? I've seen it mostly `Hyperspace`, one capital letter, `H`.